### ISSUE TITLE
Sublime stops responding when I attempt to access the /unit tests folder in an Ember project.

### ISSUE THREAD

**Issue by sheriffderek** [borela/naomi#17]
Sublime stops responding when I attempt to access the /unit tests folder in an Ember project.

To recreate the scenario, you'd install ember-cli and create a new project + generate a model for something, (in my case 'thing') - which creates a /unit/thing-test.js + then just try and access the unit folder with fuzzy search or with the mouse.

Keywords could be "unit" or "thing" - I have no idea how to debug this, but I really loved the syntax highlighting. :)

**Comment by borela** [comment]
Hi 
I am glad the syntax is being useful, nice find, I am surprised this bug didn't bite me too, it was caused by the globals `modules`. I sent a hotfix to the master branch, can you check if the fix is working for you too?

**Comment by sheriffderek** [comment]
If I just _install Naomi_ will that grab the hotfix? If so, then something is still wrong. Sublime freezes - but only on /tests/unit/*  - _if not_ how to I specify the branch to install?

**Comment by borela** [comment]
No, you have to [clone on the packages directory](https://github.com/borela/naomi#manual-installation-recommended-if-you-want-faster-updates), I didn't release it yet so package control will not update your package.

**Comment by sheriffderek** [comment]
OK! I ran through the same set of actions - and NO FREEZE. : ) HOORAY!

How did you track that down with my limited info?

**Comment by borela** [comment]
I created the ember project and opened the source code that caused the crash(the unit test) in notepad, after that I tried typing each word manually until a I hit the point that froze sublime.

I repeated the test and found that if I type `F` after the `module` it was going into a infinite loop somehow. Then I disabled the [globals file](https://github.com/borela/naomi/blob/master/syntaxes/fjsx15/global.sublime-syntax) and error vanished.

Looking at the file I found that it was consuming too much, there were no bounds on some words `\b` and that caused the backtracking which turned into a infinite loop. 

Finding these errors is also why I broke the syntax into a ton of files, it's easier to guess the state of the engine.
